contrib/docker/README.md

```diff
@@ -0,0 +1,115 @@
+# Dockerfile for end users
+
+For users who take comfort in containerization.
+
+## Benefits
+
+- Distro agnostic: run on any Linux distro with docker installed
+- Security isolation: run untrusted software without fear
+- Resource isolation: run oxend with a specific amount of CPU and memory
+- Simplicity: get up and running with only a few commands
+
+## Quickstart
+
+If you just want to build the container image and get oxend running:
+
+```
+$ sh build_container.sh
+$ docker run -d -p 22022:22022 --name oxend oxend:8.1.6
+```
+
+The blockchain and wallet won't be persisted beyond the containers lifetime, so you probably want to continue reading if
+you want to run the container long-term.
+
+## Usage details
+
+### Building the container
+
+`build_container.sh` builds a Docker container image based on Ubuntu 20.04 using the most-recent oxen pre-compiled binaries.
+The `RELEASE` variable can be updated for any release version. Once built, the container image is stored locally:
+
+```
+$ docker images
+REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
+oxend               8.1.6               d3cafa2081b0        9 days ago          185MB
+```
+
+The image can be used to run the oxend container locally. The image can also be uploaded to a docker registry so that
+other users can use it without needing to build it.
+
+### Run the container
+
+Run the container, storing the blockchain in Docker volume `oxen-blockchain`:
+
+`$ docker run --name oxend -d --mount "source=oxen-blockchain,target=/home/oxen/.oxen,type=volume" oxend:8.1.6`
+
+Same as above, but additionally mounting the wallet directory `/home/<your_user>/Oxen` to `/wallet` in the container:
+
+```
+$ docker run --name oxend -d --mount "source=oxen-blockchain,target=/home/oxen/.oxen,type=volume" \
+                             --mount "source=/home/<your_user>/Oxen,target=/wallet,type=bind" oxend:8.1.6
+```
+
+Check on how the synchronization is going:
+
+```
+$ docker logs oxend
+```
+
+#### Limiting resource usage
+
+On a shared server, it may be useful to limit the amount of resources the oxend container can consume. The recommended specs are 
+4G memory, 2 CPUs, and 30G disk. The memory and CPUs can be limited easily in the `run` command:
```

Comment:
The memory is fine, but if you allocate more cores you will get a much faster blockchain sync (unless you are on a very slow connection).